openslides/motions/config_variables.py

```diff
@@ -30,154 +29,154 @@ def get_config_variables():
-        subgroup=ugettext_lazy('General'))
+        group='Motions',
+        subgroup='General')
 
     yield ConfigVariable(
         name='motions_identifier',
         default_value='per_category',
         input_type='choice',
-        label=ugettext_lazy('Identifier'),
+        label='Identifier',
         choices=(
-            {'value': 'per_category', 'display_name': ugettext_lazy('Numbered per category')},
-            {'value': 'serially_numbered', 'display_name': ugettext_lazy('Serially numbered')},
-            {'value': 'manually', 'display_name': ugettext_lazy('Set it manually')}),
+            {'value': 'per_category', 'display_name': 'Numbered per category'},
+            {'value': 'serially_numbered', 'display_name': 'Serially numbered'},
+            {'value': 'manually', 'display_name': 'Set it manually'}),
         weight=315,
-        group=ugettext_lazy('Motions'),
-        subgroup=ugettext_lazy('General'))
+        group='Motions',
+        subgroup='General')
 
     yield ConfigVariable(
         name='motions_preamble',
-        default_value=_('The assembly may decide,'),
-        label=ugettext_lazy('Motion preamble'),
+        default_value='The assembly may decide,',
+        label='Motion preamble',
         weight=320,
-        group=ugettext_lazy('Motions'),
-        subgroup=ugettext_lazy('General'),
+        group='Motions',
+        subgroup='General',
         translatable=True)
 
     yield ConfigVariable(
         name='motions_stop_submitting',
         default_value=False,
         input_type='boolean',
-        label=ugettext_lazy('Stop submitting new motions by non-staff users'),
+        label='Stop submitting new motions by non-staff users',
         weight=325,
-        group=ugettext_lazy('Motions'),
-        subgroup=ugettext_lazy('General'))
+        group='Motions',
+        subgroup='General')
 
     yield ConfigVariable(
         name='motions_allow_disable_versioning',
         default_value=False,
         input_type='boolean',
-        label=ugettext_lazy('Allow to disable versioning'),
+        label='Allow to disable versioning',
         weight=330,
-        group=ugettext_lazy('Motions'),
-        subgroup=ugettext_lazy('General'))
+        group='Motions',
+        subgroup='General')
 
     # Amendments
     # Amendments currently not implemented. (TODO: Implement it like in OpenSlides 1.7.)
     yield ConfigVariable(
         name='motions_amendments_enabled',
         default_value=False,
         input_type='boolean',
-        label=ugettext_lazy('Activate amendments'),
+        label='Activate amendments',
         hidden=True,
         weight=335,
-        group=ugettext_lazy('Motions'),
-        subgroup=ugettext_lazy('Amendments'))
+        group='Motions',
+        subgroup='Amendments')
 
     yield ConfigVariable(
         name='motions_amendments_prefix',
         default_value=pgettext('Prefix for the identifier for amendments', 'A'),
```

Comment:
remove the pgettext and just use the string "A" als default_value.
On JavaScript site where you mark 'A' translateable try to add the context 'Prefix for the identifier for amendments' to the gettext function that translators understand what 'A' should be.